src/ReleaseHistory.md

```diff
@@ -1,5 +1,8 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+* FEATURE: Improve `FileRegionsCache` testability [#2131](https://github.com/microsoft/sarif-sdk/pull/2131)
```

Comment:
I'm not sure I see this change as significant as to require a readme note (particularly since marking a non-virtual method as virtual is a non-breaking change. If we start to simply document each merge, maybe we should be handling this via descriptive commit messages (and letting the GH release process provide our docs. Let's discuss with Chris at Monday's stand-up.

Reply:
Yeah, you are right. Let me revert! Thanks for reviewing!